ncm-metaconfig/src/main/metaconfig/ptpd/pan/schema.pan

```diff
@@ -0,0 +1,76 @@
+declaration template metaconfig/ptpd/schema;
+
+@documentation{
+    PTPv2 configuration based on 2.3.1 configfile
+}
+# generated based on
+# grep -B 1 'ptpengine:' /etc/ptpd2.conf | sed "s/^;\?ptpengine:\(.*\)=/    '\1' ? string # default /; s/^; *\(.*\)/    @{\1}/" | grep -v '\--'
+
+include 'pan/types';
+
+# an interface with an ip
+# TODO move to core templates
```

Comment:
well, i'll actualy solve the issue. TODOs that are not tracked indeed never get done.